lib/ansible/modules/monitoring/_logicmonitor.py

```diff
@@ -41,6 +41,7 @@
   - LogicMonitor is a hosted, full-stack, infrastructure monitoring platform.
   - This module manages hosts, host groups, and collectors within your LogicMonitor account.
 version_added: "2.2"
+deprecated: Deprecated in 2.4. Use M(logicmonitor_device) instead.
```

Comment:
since these modules apply to multiple resource types, we are replacing with new modules per resource, so there isn't technically a one-to-one replacement for this module. I'm not sure if there's a better way to indicate that.